_client/src/Ui/MainWindow.cpp

```diff
@@ -0,0 +1,30 @@
+#include "MainWindow.hpp"
+
+MainWindow::MainWindow(QWidget *parent)
+    : QMainWindow(parent)
+{
+    init();
+}
+
+void MainWindow::init()
+{
+    // set size
+    this->resize(MainWidSize::WINDOW_WIDTH, MainWidSize::WINDOW_HEIGHT);
```

Comment:
init method exist only for initialization purposes, do this stuff in different place

I recommend to create different method `setupUi`

Reply:
created setupUI